src/test/java/pageObjects/DevicePageTests.java

```diff
@@ -13,19 +14,62 @@ public void successfullyCreateDevice() {
         Assert.assertTrue(driver.findElement(By.id("device-table-row-0")).isDisplayed());
     }
 
+    @Test
+    public void successfullyEditADeviceName() {
+        devicePage.successfullyEditADeviceName();
+        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+        Assert.assertEquals(successEdit, "Item updated successfully!");
```

Comment:
Could you check if the name of the device was updated by adding another assertion?